pkg/syncer/sync.go

```diff
@@ -78,7 +78,8 @@ func (s *Syncer) SyncPendingCharts(names ...string) error {
 		}
 		var packagedChartPath string
 
-		if s.relocateContainerImages {
+		intermediateScenario := s.source.GetIntermediateBundlesPath() != "" || s.target.GetIntermediateBundlesPath() != ""
```

Comment:
I'd add a comment explaining that this means that we are running a partial sync, repo => intermediate or intermediate => repo